continuous-integration/use-cases/markmap.md

```diff
@@ -0,0 +1,65 @@
+* Continuous Integration
+    * For Analysis and Testing
+        * For verification, notification and assembly
+            * [Git hooks](https://git-scm.com/docs/githooks)
+            * [Github Actions](https://github.com/features/actions)
+            * Build tooling (e.g. [Maven plugins](https://maven.apache.org/plugins/index.html), [SetupTools](https://github.com/pypa/setuptools), [Make](https://www.gnu.org/software/make/))
+            * [Checksum hashing](https://en.wikipedia.org/wiki/Hash_function)
+        * For credentialing
+            * keystore ([Jenkins Credentials Binding Plugin](https://plugins.jenkins.io/credentials-binding/), et. al.) [&#x1F3C1; STARTER KIT](../starter-kits/#jenkins-project-setup-and-configuration)
```

Comment:
For the starter kits that don't exist yet, please add a (TBD) - we shouldn't have dead-links.

Reply:
e.g. "STARTER KIT (TBD)" without the HREF.

Reply:
Done.